sys/conf/kern.pre.mk

```diff
@@ -50,6 +50,10 @@ NM?=		nm
 OBJCOPY?=	objcopy
 SIZE?=		size
 
+.if ${MACHINE_ABI:Mpurecap}
```

Comment:
I'd be happier to just leave this commit off of dev for now.  I will probably get all the fixes for modules merged to dev by the time RISC-V purecap kernels are built anyway, and this just adds more work to revert later if we keep it.

Reply:
Dropped.